docs/documentation/zh/declaration-files/By Example.md

```diff
@@ -231,21 +228,20 @@ declare var foo: number;
 
 ## Global Functions
 
-_Documentation_
+_文档_
 
-> You can call the function `greet` with a string to show a greeting to the user.
+> 你可以用一个字符串调用函数`greet`来向用户显示问候语。
 
-_Code_
+_代码_
 
 ```ts
 greet("hello, world");
 ```
 
-_Declaration_
+_声明_
```

Comment:
```suggestion
*声明*
```

Reply:
I don't see this one and all the similar ones convention in https://github.com/ts-zh-docs/TypeScript-Website/blob/v2/docs/translation-guide.md, could you point the related rule?